### DISCUSSION TITLE
Player Ownership %

### DISCUSSION THREAD

**Discussion by benwolbransky** [cwendt94/espn-api#358] [Ideas]
I'd love if I could pull ownership % of a given player. Is that possible?

**Comment by cwendt94** [comment]
Could you give me an example, not sure what the ownership % means.

**Comment by benwolbransky** [comment]
On ESPN , you can see the % ownership a player has.

On a players profile page, it shows the players position rank, his average
points per game, and in what percentage of leagues he is rostered.

**Reply by cwendt94** [reply]
Oh, got it! Yeah I can definitely add that to the player class!

**Comment by benwolbransky** [comment]
What a huge help, thank you!
